charts/kubernetes/values.yaml

```diff
@@ -135,6 +135,7 @@ scraper:
         count: 10
 
 playbooks:
+  enabled: true
```

Comment:
@adityathebe @moshloop 

Should we not breakdown the playbooks by functionality ?

Like:
```
playbooks:
  kubernetesDeletePod: 
  requestNamespaceAccess:
  ```